atomsci/ddm/utils/datastore_functions.py

```diff
@@ -115,11 +122,11 @@ def initialize_model_tracker(new_instance=False):
     if not clients_supported:
         raise Exception("Model tracker client not supported in current environment.")
 
-    if not 'MLMT_REST_API_URL' in os.environ:
+    if 'MLMT_REST_API_URL' not in os.environ:
         os.environ['MLMT_REST_API_URL'] = 'https://twintron-blue.llnl.gov/atom/mlmt/api/v1.0/swagger.json'
 
     # MLMT service uses same API token as datastore. Make sure it gets set in the environment.
-    ds_client = config_client()
+    _ds_client = config_client()
```

Comment:
does it make sense to configure datastore client just to set the token for MLMT client?